sites/perioimplantadvisory.com/config/gam.js

```diff
@@ -1,27 +1,6 @@
-const GAMConfiguration = require('@base-cms/marko-web-gam/config');
+const configureGAM = require('@endeavor-business-media/package-shared/config/gam');
 
-const config = new GAMConfiguration('21687441225', { basePath: 'PIA' });
-
-config
-  .setTemplate('LB1', {
-    size: [[970, 250], [970, 90], [970, 66], [728, 90], [320, 50], [300, 50], [300, 100]],
-    sizeMapping: [
-      { viewport: [980, 0], size: [[970, 250], [970, 90], [970, 66], [728, 90]] },
-      { viewport: [750, 0], size: [728, 90] },
-      { viewport: [320, 0], size: [[300, 50], [320, 50], [300, 100]] },
-    ],
-  })
-  .setTemplate('LB2', {
-    size: [[970, 90], [970, 66], [728, 90], [320, 50], [300, 50]],
-    sizeMapping: [
-      { viewport: [980, 0], size: [[970, 90], [970, 66], [728, 90]] },
-      { viewport: [750, 0], size: [728, 90] },
-      { viewport: [320, 0], size: [[300, 50], [320, 50]] },
-    ],
-  })
-  .setTemplate('RAIL1', { size: [300, 250] })
-  .setTemplate('RAIL2', { size: [300, 600] })
-  .setTemplate('LM', { size: [[300, 250], [300, 600]] });
+const config = configureGAM({ basePath: 'PIA' });
```

Comment:
Since this is using the dental layout (cards only), this config should override the `RAIL2` and `LM` templates to use 300x250 only. See DE for an example of this.